jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -249,7 +253,11 @@ def _parse_hdulist(app, hdulist, file_name=None,
         if hdu.name != 'PRIMARY' and 'PRIMARY' in hdulist:
             metadata[PRIHDR_KEY] = standardize_metadata(hdulist['PRIMARY'].header)
 
-        sc = _return_spectrum_with_correct_units(flux, wcs, metadata, data_type, hdulist=hdulist)
+        if not is_big_cube:
+            sc = _return_spectrum_with_correct_units(
+                flux, wcs, metadata, data_type, hdulist=hdulist)
+        else:
+            sc = Spectrum1D(flux=flux, wcs=wcs, meta=metadata)
```

Comment:
will this also skip any necessary units stuff?  Can we just set/pass `hdulist=None` and still use `_return_spectrum_with_correct_units` or is that still too expensive?

Reply:
`_return_spectrum_with_correct_units` creates a new `Spectrum1D` object. I have not looked into whether it does a copy or what. I was just trying some possible low hanging fruits to see if anything helps. Given the ticket is only 2 points, I didn't dig too deep here. There is a chance we do not have to disable this (and I don't think we want to disable if we can avoid it).